src/evict/evict_inline.h

```diff
@@ -284,7 +275,7 @@ __wti_evict_hs_dirty(WT_SESSION_IMPL *session)
     bytes_max = conn->cache_size;
 
     return (__wt_cache_bytes_plus_overhead(cache, __wt_atomic_load64(&cache->bytes_hs_dirty)) >=
-      ((uint64_t)(cache->eviction_dirty_trigger * bytes_max) / 100));
+      ((uint64_t)(conn->evict->eviction_dirty_trigger * bytes_max) / 100));
```

Comment:
nit:
```suggestion
    return (__wt_cache_bytes_plus_overhead(cache, __wt_atomic_load64(&S2C(conn)->bytes_hs_dirty)) >=
```
and we don't need the local `cache` variable

Reply:
but cache is used twice here?

Reply:
My bad, must've missed it 👀